racker/Minerva
# Time range mnrva 759

## JIRA:
See ticket here - https://jira.rax.io/browse/MNRVA-759

 ## Description:
 Add Time Range Component to Visualize Module.

 - Added DateTime Picker https://daniel-projects.firebaseapp.com/owlng/date-time-picker 
 - Added `stylePreprocessorOptions` to _angular.json_
 - Update VisualizePageComponent to subscribe to URL route params
 
 ## Testing:
 1. Visualize Page loads
 2. Change duration radio selection - URL params should update
 3. Change to Custom range - text boxes will appear
 4. Enter calendar dates - apply button will enable

 ## Screenshots:
![timeComponent](https://user-images.githubusercontent.com/5041718/119434213-554f8080-bcdd-11eb-878b-a38c96dd9881.gif)
